WebNetConsole/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using HttpClient client = new();
+
+client.DefaultRequestHeaders.Accept.Clear();
+client.DefaultRequestHeaders.Add("User-Agent", "EPAM WEB .NET Mentoring Program");
+
+await ProcessRepositoriesAsync(client);
+
+static async Task ProcessRepositoriesAsync(HttpClient client)
+{
+    var categoriesJson = await client.GetStringAsync(
+    "https://localhost:7065/api/Categories/");
```

Comment:
Just an advice: use _lowercase_ for your URLs since this is a correct way for working with RESTful web-service, also, you can easily put `https://localhost:7065` into a constant and reuse for both categories and products